### DISCUSSION TITLE
LoRaWAN Application

### DISCUSSION THREAD

**Discussion by Manjunath2001-debug** [danak6jq/Seeed-LoRa-E5#30] [General]
Hi, I have some issues in this lora-e5 github code ?
1)I am trying to change the parameters like deveui,joineui and appkey and region , In this i am calling my own function and changing these para from main function in your code ?
2)How to add our application to your github code ?
3)If i pass parameters from main function it should change the keys in stack ?

please provide some idea to do this job.
Thanks in advance

**Comment by danak6jq** [comment]
The example code in this repo is directly taken from the ST example code, I suggest referring the ST documentation for the example
